xblockmufi/templates/view.html

```diff
@@ -0,0 +1,18 @@
+<div class="xblockmufi_block">
+    <h2>{{ display_name }}</h2>
+    <div class='your_answer nodisplay'>{{ your_answer_label }}</div>
+    <div class="xblockmufi_block_textarea txtEditor">
+        {% include "_text_editor.html" %}
```

Comment:
Nice, although maybe the filename `_text_editor` I told you to give it doesn't make the most sense...

Maybe we could change it to: `_text_editor_toolbars`?